hooks/600-no-debian.chroot

```diff
@@ -54,4 +54,10 @@ rm -f var/log/bootstrap.log \
 # also remove obsolete cron jobs
 rm -f /etc/cron.daily/dpkg \
       /etc/cron.daily/passwd
-    
+# and services from dpkg
+rm -f var/lib/systemd/deb-systemd-helper-enabled/dpkg-db-backup.timer.dsh-also \
+      var/lib/systemd/deb-systemd-helper-enabled/timers.target.wants/dpkg-db-backup.timer \
+      usr/libexec/dpkg/dpkg-db-backup \
```

Comment:
We could `rmdir usr/libexec/dpkg` after that.

Reply:
Done now